docs/docs/api-reference.md

```diff
@@ -184,6 +184,10 @@ Optional. Set to `true` if you want to instruct Rune to delay showing of the gam
 
 Returns the amount of milliseconds that have passed since the start of the game. See [Using Time in your Game](advanced/real-time-games.md#game-time).
 
+### `Rune.worldTime()` {#runeworldtime}
+
+Returns the amount of milliseconds since the start of epoch. See [Using Time in your Game](advanced/real-time-games.md#world-time).
```

Comment:
```suggestion
Returns the amount of milliseconds since the start of epoch with a precision of 1 second. See [Using Time in your Game](advanced/real-time-games.md#world-time).
```